tests/__snapshots__/api.test.ts.snap

```diff
@@ -97,7 +101,8 @@ exports['api explore should generate data when provided with file with separated
         "src/bar.js": 104,
         "src/foo.js": 97,
         "<unmapped>": 1
-      }
+      },
+      "filesCoverage": {}
```

Comment:
We should not add `filesCoverage` property unless coverage file specified

Reply:
I feel that would make the code more complicated without any objective benefit, it seems like I'd just have to add a bunch of runtime type checking for no real reason. How strongly do you feel about this one?

Reply:
Values like `{}` break checks like `if(filesCoverage)`. If there are no data not property should be added. Or at least it should have falsy value

Reply:
I would say don't do checks like that then, instead do `if(Object.keys(filesCoverage).length>0)`, or a helper function can be used to check if there's `filesCoverage` to abstract that away.

I've gone ahead and changed it to be optional as you request, which does add some boilerplate to safely access it.